### DISCUSSION TITLE
Are the labels in the target polygon dataset missing?

### DISCUSSION THREAD

**Discussion by EY-glowglowglow** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#44] [4.0 Exploring the data and building models]
This is a real world 'messy' data set. Many of the linescan images do not have annotations provided, and for some that do, it can be challenging to match them to the correct annotation set. Part of the challenge involves identifying what to do with these to overcome the issue.

**Comment by mlds2020** [comment]
Has anyone else found that only 37 out of the 129 training images in the "resources/challenge1_train.csv" have respective polygons in the "resources/fire_boundaries.shp" file? Or am I missing something major?

**Reply by EY-glowglowglow** [reply]
Hi there, as mentioned above, this is a real world dataset. And hence, part of the challenge is to come up with ways to overcome this limitation. Good luck!

**Reply by ImonEmmanuel** [reply]
So do you suggest we drop them ??? 


Just like a conventional ML problem Preprocessing technique?

**Reply by mlds2020** [reply]
> Hi there, as mentioned above, this is a real world dataset. And hence, part of the challenge is to come up with ways to overcome this limitation. Good luck!

Yes, I understood that - I was only a little surprised that 129 training images were provided of which only 28% are labeled, so I wanted to make sure I am not overlooking anything major! Thanks!


> So do you suggest we drop them ???
> 
> Just like a conventional ML problem Preprocessing technique?

That's what I am doing... Unlabeled data is no use for supervised learning. If you wanted to go really hard I guess you could manually label some of the data, but I don't think that's the intention of a data science competition, and it will likely disqualify you from winning in the final stage of the competition...

**Reply by EY-glowglowglow** [reply]
Hi there,

Around 50% have labels available, they are just not directly available via an easy matching mechanism (i.e. the label/annotation is not easy to match to the correct image). **This is outlined in the Challenge 1 getting started notebook.** 
Possible matching techniques are identifying labels that occurred at a similar time and place to an image, or using regex techniques to extract information from the metadata on the labels (e.g. the source name field) to match to the image name.

Hope that helps!
